tests/integration/individual_response/test_individual_response.py

```diff
@@ -21,6 +21,10 @@ def individual_response_link(self):
         if response_paragraph:
             return response_paragraph.find_next()["href"]
 
+    def get_next_location_url_from_interstitial_page(self):
+        submit_button = self.getHtmlSoup().find("a", {"data-qa": "btn-submit"})
+        return submit_button.attrs["href"]
```

Comment:
This method should be consistent with the `individual_section_link` and `individual_response_link` methods. It should be defined as a `@property` and named something like `individual_response_start_link`.